docs/tutorial.md

```diff
@@ -0,0 +1,369 @@
+        return size;
+    }
+
+    public void setSize(Integer size) {
+        this.size = size;
+    }
+}
+```
+
+#### `MemcachedStatus`
+
+Similar to the Spec file `MemcachedStatus` file got scaffolded as part of the `create api` command. The user has to modify a Status file. For the Memcached example, we too list of nodes as shown below.
+The nodes field is a list of string values and it contains the name of the Memcached pods.
+
+
+```
+import java.util.ArrayList;
+import java.util.List;
+
+// MemcachedStatus defines the observed state of Memcached
+public class MemcachedStatus {
+
+    // Add Status information here
+	// Nodes are the names of the memcached pods
+    private List<String> nodes;
+
+    public List<String> getNodes() {
+        if (nodes == null) {
+            nodes = new ArrayList<>();
+        }
+        return nodes;
+    }
+
+    public void setNodes(List<String> nodes) {
+        this.nodes = nodes;
+    }
+}
+```
+
+**Note** The Node field is just to illustrate an example of a Status field. In real cases, it would be recommended to use [Conditions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties).
+
+#### `Memcached`
+
+`Memcached` file scaffolded via `create api` command. It extends the properties of `MemcachedSpec` and `MemcachedStatus`.
+
+```
+// Memcached is the Schema for the memcacheds API
+
+@Version("v1alpha1")
+@Group("cache.example.com")
+public class Memcached extends CustomResource<MemcachedSpec, MemcachedStatus>
+    implements Namespaced {}
+```
+
+### Apply CR and CRD's using below command
+
+#### CRD - Custom Resource
+
+Create a file with the name `crd.yaml`. CRD enables users to add their own/custom objects to the Kubernetes cluster.
+
+This file will contain the below content. 
+
+```
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  name: memcacheds.cache.example.com
+spec:
+  group: cache.example.com
+  names:
+    kind: Memcached
+    listKind: MemcachedList
+    plural: memcacheds
+    singular: memcached
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        ...
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+```
+
+Create the CRD:
+
+`kubectl apply -f cr.yaml`
+
+### Create Memcached Custom Resource - memcached-sample.yaml
+
+Update the sample Memcached CR manifest at k8s/samples/memcached-sample.yaml and define the spec as the following
+
+This file will contain the below content. 
+
+```
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-sample
+spec:
+  # Add fields here
+  size: 1
+```
+
+Create the CR:
+
+`kubectl apply -f memcached-sample.yaml`
+
+## Implement the Controller
```

Comment:
This section is better but I don't know what file I'm supposed to be editing. And I don't know if I'm supposed to be adding the examples you've posted in this section. See more comments below.

Reply:
Modified the Controller section by adding above information.